radis/lbl/broadening.py

```diff
@@ -887,27 +887,37 @@ def _check_accuracy(self, df, wstep):
 
         WARN_THRESHOLD = 3
         ERROR_TRESHOLD = 1
+        wstep_auto = self.wstep_auto
         if wstep > min_width / ERROR_TRESHOLD:
-            self.warn(
-                f"Some lines are too narrow (FWHM ~ {min_width:.2g} cm⁻¹) for "
-                + f"the current spectral grid (wstep={wstep}). Please reduce "
-                + f"wstep to (at least) below {min_width/ERROR_TRESHOLD:.2g} cm⁻¹ "
-                + f"or (suggested) {min_width/WARN_THRESHOLD:.2g} cm⁻¹. "
-                + "You can also ignore by setting `warnings={'AccuracyError':'ignore'}` "
-                + "(if you know what you're doing!)",
-                "AccuracyError",
-            )
+            if wstep_auto:
+                self.params.wstep = min_width / WARN_THRESHOLD
```

Comment:
It's confusing that this very important step is done in the function `_check_accuracy`. 
Can we move it further up ? For instance in the method that builds the wavegrid array.  Keep only the warnings in the `_check_accuracy` method. 

You'll need "WARN_THRESHOLD" as some kind of global parameter. Temporarily you can use [radis/params.py](https://github.com/radis/radis/blob/develop/radis/params.py) for that. We'll move them in a .json/.yaml later on.